frontend/src/components/common/LogComponent.tsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/react-in-jsx-scope */
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Select,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+
+type LogComponentProps = {
+  employee: string;
+};
+
+// eslint-disable-next-line react/prop-types
+const LogComponent: React.FC<LogComponentProps> = ({ employee }) => {
+  const [building, setBuilding] = useState<string>("");
+  const [tags, setTags] = useState<string>("");
+
+  const handleBuildingChange = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    setBuilding(event.target.value);
+  };
+
+  const handleTagsChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTags(event.target.value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // eslint-disable-next-line no-console
+    console.log(
+      `Employee: ${employee}, Date: ${new Date()}, Building: ${building}, Tags: ${tags}`,
+    );
+  };
+
+  return (
+    <div className="log-entry">
+      <h1>New Log Entry Details</h1>
+      <form onSubmit={handleSubmit}>
+        <FormControl>
+          <FormLabel>Employee</FormLabel>
+          <Input type="text" value={employee} isDisabled />
```

Comment:
would you be able to mock an employee name for now? Just want to make sure it renders correctly with appropriate styling